rtt-target/src/init.rs

```diff
@@ -17,17 +17,17 @@ macro_rules! rtt_init_channels {
     (
         $field:expr;
         $number:literal: {
-            size: $size:literal
-            $( mode: $mode:ident )?
-            $( name: $name:literal )?
+            size: $size:expr
+            $(, mode: $mode:path )?
+            $(, name: $name:literal )?
```

Comment:
```suggestion
            $(, name: $name:literal )?
            $(,)?
```

I meant like this :)

Reply:
got it! thanks!